accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/validators/impl/AuthorizableValidatorImpl.java

```
@@ -36,16 +37,58 @@ public void validate(final AuthorizableConfigBean authorizableConfigBean)
 
     private boolean validate() throws AcConfigBeanValidationException {
         if (enabled) {
-            return validateMemberOf(this.authorizableConfigBean)
+            return validateAuthorizableProperties(this.authorizableConfigBean)
+                    && validateMemberOf(this.authorizableConfigBean)
                     && validateMembers(this.authorizableConfigBean)
                     && validateAuthorizableId(this.authorizableConfigBean);
         }
         return true;
     }
 
+    public boolean validateAuthorizableProperties(

```

Introduced validation for some requirements to fail early